src/settings/components/AnonymizingTypeSelect/AnonymizingTypeSelectContainer.js

```diff
@@ -94,7 +94,7 @@ class AnonymizingTypeSelectContainer extends Component {
         <Field
           name={`${path}Selected`}
           component={({ meta }) => {
-            return meta.touched && <span className={css.error}>{meta.error}</span>;
+            return meta.touched && !!meta.error && <span className={css.error}>{meta.error}</span>;
```

Comment:
```suggestion
            return meta.touched && meta.error && <span className={css.error}>{meta.error}</span>;
```

Reply:
Thank you for your comments, Victor. But in this case, since the meta.error by default is undefined, explicit conversion to a boolean is required to avoid rendering an undefined value.

Reply:
Please see the updated comment. I think it should avoid extra thinking about this converting stuff and thus simplify things from the code readability standpoint.

Reply:
Yes. Agree. This is better in terms of code readability.

Reply:
Thank you.